typescript/packages/lookslike-high-level/src/recipes/articlesIframe.tsx

```diff
@@ -0,0 +1,82 @@
+import { h } from "@commontools/html";
+import { recipe, UI, NAME } from "@commontools/builder";
+import type { JSONSchema } from "@commontools/builder";
+
+//<TYPE>IFRAME V0</TYPE>
+
+type Recipe = {
+  type: "iframe";
+  src: string;
+  argumentSchema: JSONSchema;
+  resultSchema: JSONSchema;
+  spec: string;
+  name: string;
+};
+
+//<SRC>
+// const src = "<p>hello world</p>";\
+import src from "./articlesIframe.html?raw";
```

Comment:
This is just an example so I could play with different schema shapes, can be dropped for merge.